src/node.h

```diff
@@ -528,22 +528,22 @@ NODE_EXTERN void AddPromiseHook(v8::Isolate* isolate,
 /* Returns the id of the current execution context. If the return value is
  * zero then no execution has been set. This will happen if the user handles
  * I/O from native code. */
-NODE_EXTERN async_uid AsyncHooksGetCurrentId(v8::Isolate* isolate);
+NODE_EXTERN async_uid AsyncHooksGetExecutionAsyncId(v8::Isolate* isolate);
 
-/* Return same value as async_hooks.triggerId(); */
-NODE_EXTERN async_uid AsyncHooksGetTriggerId(v8::Isolate* isolate);
+/* Return same value as async_hooks.triggerAsyncId(); */
+NODE_EXTERN async_uid AsyncHooksGetTriggerAsyncId(v8::Isolate* isolate);
```

Comment:
Maybe just one extra thought, if you want to do this in a ABI/API-compatible way, you could just add
```c++
// legacy
#define AsyncHooksGetExecutionAsyncId AsyncHooksGetCurrentId
#define AsyncHooksGetTriggerAsyncId AsyncHooksGetTriggerId
```
before these definitions